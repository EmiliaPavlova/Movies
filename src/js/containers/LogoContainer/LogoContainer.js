import React, { Component } from 'react';
import Logo from '../../components/Logo/Logo';

class LogoContainer extends Component {
  render() {
    return <Logo text='netflixroulette' />
  }
}

export default LogoContainer;
