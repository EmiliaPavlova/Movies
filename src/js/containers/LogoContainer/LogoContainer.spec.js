import React from 'react';
import { mount } from 'enzyme';
import LogoContainer from './LogoContainer';

const mountLogoContainer = props => mount(<LogoContainer {...props} />);

describe('LogoContainer', () => {
  describe('Snapshot', () => {
    it('matches the snapshot', () => {
      const wrapper = mountLogoContainer();
      expect(wrapper).toMatchSnapshot();
    })
  });
});
