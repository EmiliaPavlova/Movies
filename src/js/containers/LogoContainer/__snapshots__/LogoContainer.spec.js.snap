// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoContainer Snapshot matches the snapshot 1`] = `
<LogoContainer>
  <Logo
    text="netflixroulette"
  >
    <div
      className="logo"
    >
      netflixroulette
    </div>
  </Logo>
</LogoContainer>
`;
